apps/auth_content/templates/auth_content/big_cities_home_page.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'hip/static_page.html' %}
+
+{% block sidebar %}
+  {% include 'includes/sidebar.html' %}
+{% endblock %}
+
+{% block page_css_class_class %}bigcities-homepage-hip{% endblock %}
```

Comment:
Guess I didn't notice this before, but why does this block have 2 `class`es in it?

Reply:
Good question. I'd like to know as well. I'll change it to have just 1 `class`